src/header/Header.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as React from "react";
+import {ReactNode} from "react";
+import {classNames} from "../utils";
+import {Icon} from "../icon";
+
+// Props for Header component
+type HeaderProps = {
+    className?: string;
+    style?: any;
+    children?: ReactNode | ReactNode[];
+};
+
+export const HeaderColor = {
```

Comment:
Export this as enum and make constant names capital.

Reply:
Done